packages/wonder-blocks-labeled-field/src/components/error-helper-text.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import WarningCircle from "@phosphor-icons/core/bold/warning-circle-bold.svg";
+import {PhosphorIcon} from "@khanacademy/wonder-blocks-icon";
+import {HelperText} from "./helper-text";
+import theme from "../theme";
+
+type Props = {
+    message?: React.ReactNode;
+    id?: string;
+    testId?: string;
+    show: boolean;
```

Comment:
There is a mandatory `show` prop for the `ErrorHelperText` component because even if there is no error, we want this component to be mounted so that the aria-live region is ready for when there is an error. Open to other suggestions on how to handle a persistent aria-live region!